test/features/interlinks/interlinks.test.js

```diff
@@ -0,0 +1,34 @@
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+import { waitForElement } from '../../helpers/waitfor.js';
+
+const { default: init } = await import('../../../libs/features/interlinks.js');
+
+describe('Interlinks', () => {
+  it('Interlinks works for english language', async () => {
+    document.body.innerHTML = await readFile({ path: './mocks/body_en.html' });
+    await init('/test/features/interlinks/mocks/keywords.json', 'en');
+    await waitForElement('a');
+    const anchorElem = document.querySelector('a');
+    expect(anchorElem.getAttribute('href')).equals('http://oceandecade.org/');
```

Comment:
`'http://oceandecade.org/'` is a good candidate for a constant since it is used in each test.

Reply:
Done